crates/icrate/examples/browser.rs

```diff
@@ -26,20 +26,30 @@ use objc2::{
     sel, ClassType,
 };
 
-type IdCell<T> = Box<RefCell<Option<Id<T>>>>;
+type IdCell<T> = Box<OnceCell<Id<T>>>;
 
 macro_rules! idcell {
+    ($name:ident => $this:expr) => {
+        $this.$name.set($name).expect(&format!(
+            "ivar should not already be initialized: `{}`",
+            concat!(stringify!($name))
```

Comment:
Nit, `concat` not required. Same below.

```suggestion
            stringify!($name)
```